config/foreman.hiera/tuning/common.yaml

```diff
@@ -22,3 +22,6 @@ postgresql::server::config_entries:
   log_line_prefix: '%t '
   log_min_duration_statement: 1000
   log_rotation_size: 200000
+
+mongodb::server::config_data:
+  storage.wiredTiger.engineConfig.cacheSizeGB: "%{facts.kafo.scenario.custom.mongo_cache_size}"
```

Comment:
If a user hasn't upgraded to wiredTiger will this configuration option existing be a no-op for them? That is, will it break if not on wiredTiger?

Reply:
it shouldn't. Let me see if I can get an older version of server to test this. It may be tricky to get one though.

Reply:
I can't get older versions of upstream / downstream boxes to get non-wiredTiger, is there a way to do it properly?

Reply:
I found an old Satellite 6.3.5 server running mmapv1 and added the following to the custom-hiera.yaml and ran the installer. The mongo service came back up without any issues.
```
mongodb::server::config_data:
  storage.wiredTiger.engineConfig.cacheSizeGB: 2
```